packages/nvidia-container-toolkit/nvidia-container-toolkit.spec

```
@@ -25,6 +26,20 @@ Requires: %{_cross_os}shimpei
 %description
 %{summary}.
 
+%package ecs
+Summary: Files specific for the ECS variants
+Requires: %{name}
```

Since these packages shouldn't be allowed to be installed together:
```suggestion
Requires: %{name}
Conflicts: %{name}-k8s
```

They won't conflict directly but installing both would make the symlink target non-deterministic.